Apsitvarkom.Api/Controllers/CleaningEventController.cs

```diff
@@ -14,15 +14,18 @@ public class CleaningEventController : ControllerBase
     private readonly IRepository<CleaningEvent> _repository;
     private readonly IMapper _mapper;
     private readonly IValidator<ObjectIdentifyRequest> _objectIdentifyValidator;
+    private readonly IValidator<CleaningEventCreateRequest> _cleaningEventCreateRequestValdiator;
```

Comment:
you made a mistake here writing the word `Validator`.
also looking at the validator name above, we could omit the `Create` part and instead have the name as follows:
```suggestion
    private readonly IValidator<CleaningEventCreateRequest> _cleaningEventCreateValidator;
```